core/src/services/pilot.rs

```diff
@@ -0,0 +1,159 @@
+use anyhow::{Result};
```

Comment:
Removing anyhow was on my todo list for updating the client, but you all got there before me. It's leading to some of the challenges with Error Handling I think. See notes below.